crypto/src/main/java/org/web3j/crypto/transaction/type/Transaction1559.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2019 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.crypto.transaction.type;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.web3j.crypto.Sign;
+import org.web3j.rlp.RlpList;
+import org.web3j.rlp.RlpString;
+import org.web3j.rlp.RlpType;
+import org.web3j.utils.Bytes;
+import org.web3j.utils.Numeric;
+
+import static org.web3j.crypto.transaction.type.TransactionType.EIP1559;
+
+/**
+ * Transaction class used for signing 1559 transactions locally.<br>
+ * For the specification, refer to p4 of the <a href="http://gavwood.com/paper.pdf">yellow
```

Comment:
I think for 1559 transactions this should link to EIP1559 - nothing about that in the yellow paper

Reply:
Yes good catch I updated that